src/current/molt/replicator-flags.md

```diff
@@ -0,0 +1,85 @@
+| <a id="discard-delay"></a> `--discardDelay`                                                 | `start`                                             | `DURATION` | Adds additional delay in discard mode; useful for gauging the impact of changefeed round-trip time (RTT).                                                                                                                                                                                                                       |
+| <a id="dlq-table-name"></a> `--dlqTableName`                                                | `start`, `pglogical`, `mylogical`, `oraclelogminer` | `IDENT`    | The name of a table in the target schema for storing dead-letter entries.<br><br>**Default:** `replicator_dlq`                                                                                                                                                                                                                  |
+| <a id="enable-checkpoint-stream"></a> `--enableCheckpointStream`                            | `start`                                             | `BOOL`     | Enable checkpoint streaming (use an internal changefeed from the staging table for real-time updates), rather than checkpoint polling (query the staging table for periodic updates), for failback replication. <br><br>**Default:** `false` (use checkpoint polling)                                                           |
+| <a id="enable-parallel-applies"></a> `--enableParallelApplies`                              | `start`, `pglogical`, `mylogical`, `oraclelogminer` | `BOOL`     | Enable parallel application of independent table groups during replication. By default, applies are synchronous. When enabled, this increases throughput at the cost of higher target pool usage and memory usage.<br><br>**Default:** `false`                                                                                  |
+| <a id="fetch-metadata"></a> `--fetchMetadata`                                               | `mylogical`                                         | `BOOL`     | Fetch column metadata explicitly, for older versions of MySQL that do not support `binlog_row_metadata`.                                                                                                                                                                                                                        |
+| <a id="flush-period"></a> `--flushPeriod`                                                   | `start`, `pglogical`, `mylogical`, `oraclelogminer` | `DURATION` | Flush queued mutations after this duration.<br><br>**Default:** `1s`                                                                                                                                                                                                                                                            |
```

Comment:
Just a note that for now `flushPeriod` and `flushSize` only apply to start, pglogical, oraclelogminer. Not MySQL yet.